portfolio/src/main/webapp/index.html

```diff
@@ -1,52 +1,89 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-    <div id="display">
-    <img src='images/my_photo.jpg'>
-    <div id="info">
-      <h1>Samiksha Kapoor</h1>
-      <p>
-          <h2>HEY!</h2>
-          Thanks for viewing my portfolio! Let me introduce myself... 
-          <br>
-          I am a Competitive Programmer. I always remain very passionate to learn about 
-          new technologies. Loves to solve the Real World Problems and want make a positive impact 
-          on this world. 
-          <br>
+
+<head>
+	<meta charset="UTF-8">
```

Comment:
Identation level is too much. 2 spaces is okay. If you are using tabs, set tabs to 2 spaces.